src/test/java/lt/andriaus/hangman/util/StringUtilsTest.java

```diff
@@ -29,11 +24,20 @@ void shouldReturnSet() {
     }
 
     @Test
-    void shouldReturnList() {
+    void shouldReturnCharList() {
         List<Character> wordList = Utils.StringToCharList(wordString);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.